pyproject.toml

```diff
@@ -18,13 +18,13 @@ python = "^3.7.0"
 click = "^8.0.0"
 black = "^22.1.0"
 toml = "^0.10.2"
-importlib_metadata = "^1.7.0"
+importlib_metadata = {version = ">=1.7.0,<5.0", python = "3.7"}
```

Comment:
just checking, does this say importlib_metadata should be installed iff python version is 3.7? 
That's close to correct but might as well be "python < 3.8"

Reply:
Also, is the syntax above equivalent to `importlib_metadata >= 1.7.0 ; python = "3.7"`, which is what I see for e.g. here: https://pip.pypa.io/en/stable/reference/requirement-specifiers/#examples

Reply:
sure, it can be <3.7. will change that now.

looks like it is similar (with the addition of `<5.0`), but this is a poerty-specific way of specifying it